clang/docs/ReleaseNotes.rst

```diff
@@ -193,6 +193,8 @@ Improvements to Clang's diagnostics
   under the subgroup ``-Wunsafe-buffer-usage-in-libc-call``.
 - Diagnostics on chained comparisons (``a < b < c``) are now an error by default. This can be disabled with
   ``-Wno-error=parentheses``.
+- Adds a diagnostic for out of bounds vector accesses; Errors by default for compile time statically
+  provable out of bounds vector accesses.
```

Comment:
"error by default" implies that it can be turned off, as implemented it is always an error.
```suggestion
- Adds an error diagnostic for out of bounds vector accesses; produces an error
  for compile time statically provable out of bounds vector accesses.
```